calculator.rb

```diff
@@ -0,0 +1,96 @@
+#create a calculator program that asks for user input and runs file from command line
+
+#define global scope variables
+results = nil
```

Comment:
Totally a small thing, but you named this variable `results`, when it never will be an array, it will always be one result. Consider renaming this to `result`